docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -27,27 +27,35 @@ These instructions show how to enable and resolve static analysis warnings to pr
 > [!TIP]
 > Ensure you're using the .NET 6 SDK or later for these steps. They will not work correctly in previous versions.
 
-### Set IsTrimmable
+### Enable the Roslyn analyzer
 
-Set `<IsTrimmable>true</IsTrimmable>` (in .NET 6+) in your library project. This will mark your assembly as "trimmable". Being trimmable means when your library is used in a trimmed application, the assembly can have its unused members trimmed in the final output.
+The quickest and easiest way to see trim warnings is to enable the Roslyn analyzer for trim compatibility. The Roslyn analyzer is useful for quick feedback in your IDE and catches many issues, but it's currently incomplete. It doesn't cover all trim analysis warnings, but the set of patterns it understands will improve over time to give more complete coverage. The Roslyn analyzer also isn't able to analyze the implementations of reference assemblies that your library depends on. Because the analysis is incomplete, it's important to follow the steps outlined in the [Show all warnings](#show-all-warnings-with-sample-application) to ensure that your library is fully compatible with trimming.
```

Comment:
I think we want to encourage people to set `IsTrimmable` over `EnableTrimAnalyzer`, because it will both enable the analyzer and allow the library to be trimmed if used in an application - so I think we should lead with `IsTrimmable` as the very first recommendation. Maybe we could make it more clear in the very first sentence of this section that it enables the analyzer?